javav2/example_code/sns/Readme.md

```diff
@@ -1,6 +1,52 @@
+- [Listing the phone numbers for which the users have selected to no longer receive future text messages.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/ListOptOut.java) (ListPhoneNumbersOptedOut command)
+- [Listing existing Amazon SNS subscriptions.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/ListSubscriptions.java) (ListSubscriptions command)
+- [Retrieving tags from an Amazon SNS topic.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/ListTags.java) (ListTagsForResource command)
+- [Retrieving a list of existing Amazon SNS topics.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/ListTopics.java) (ListTopics command)
+- [Sending an Amazon SNS text message.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/PublishTextSMS.java) (Publish command)
+- [Publishing an Amazon SNS topic.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/PublishTopic.java) (Publish command)
+- [Setting attributes for Amazon SNS.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SetSMSAttributes.java) (SetSMSAttributes command)
+- [Setting attributes for Amazon SNS topic.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SetTopicAttributes.java) (SetTopicAttributes command)
+- [Subscribing to an Amazon SNS email endpoint.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SubscribeEmail.java) (Subscribe command) 
+- [Subscribing to an Amazon SNS HTTPs endpoint.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SubscribeHTTPS.java) (Subscribe command) 
+- [Subscribing to an Amazon SNS lambda function.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SubscribeLambda.java) (Subscribe command) 
+- [Subscribing to an Amazon SNS text endpoint.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/SubscribeTextSMS.java) (SubscribeTextSMS command)  
+- [Removing an Amazon SNS subscription.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/Unsubscribe.java) (Unsubscribe command)  
+- [Initializing and using the example SNSMessageFilterPolicy class.](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/javav2/example_code/sns/src/main/java/com/example/sns/UseMessageFilterPolicy.java) (Apply command)  
```

Comment:
No need to add ending punctuation to link names.

Reply:
fixed